packages/my-near-wallet/src/lib/my-near-wallet.ts

```diff
@@ -159,28 +159,8 @@ const MyNearWallet: WalletBehaviourFactory<
     async verifyOwner({ message, callbackUrl, meta }) {
```

Comment:
To fix linting issues either log the `callbackUrl` and  `meta` or remove them since they are not used anyway.